websites/H/hentaifox/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "Bas1C",
+    "id": "303603391941050370"
+  },
+  "service": "hentaifox",
+  "description": {
+    "tr": "Hentai manga okuma sitesidir.",
+    "en": "Hentai is manga reading site."
```

Comment:
Try and use a more lengthy description. Look up the website in Google for instance and copy the description text.